extensions/wikia/EditPageLayout/css/EditPageLayout.scss

```diff
@@ -599,3 +599,8 @@ body {
 		@include breakpoints-large-typography;
 	}
 }
+
+#InfoboxPreview {
```

Comment:
@katanka Please address @RafLeszczynski and @pauloslund comments about not using ID (and perhaps using class name instead).